calico_node/filesystem/etc/calico/confd/config/custom_filters.cfg

```diff
@@ -0,0 +1,5 @@
+# Generated by confd
```

Comment:
You're right about (2). But this will be not an optimal solution from the performance point of view, since confd will have to watch a wider Etcd subtree that it actually needs for custom filters. That's why I used more precise keys. It may be not a problem if you don't expect a lot of changes in `/calico/bgp/v1/global/` though. So if you confirm it's OK, I'll update the patch.

As for (1) is merged. Let me update this patch and see how CI goes.